core-services/prow/02_config/_boskos.yaml

```diff
@@ -367,6 +367,10 @@ resources:
   - us-east1--gcp-quota-slice-69
   state: free
   type: gcp-quota-slice
+- max-count: 15
+  min-count: 15
+  state: free
+  type: hypershift-quota-slice
```

Comment:
Please also add this to clusters/app.ci/prow/03_deployment/boskos_reaper.yaml

Reply:
Was that supposed to happen during `make boskos-config`? It didn't. I updated `boskos_reaper.yaml` manually.